src/operator/pooling-inl.h

```diff
@@ -50,13 +48,6 @@ struct PoolingParam : public dmlc::Parameter<PoolingParam> {
     .add_enum("sum", pool_enum::kSumPooling)
     .describe("Pooling type to be applied.");
 
-    DMLC_DECLARE_FIELD(pooling_convention).set_default(pool_enum::kValid)
```

Comment:
this is include by MKL2017 implemented layers integration to improve IA perf. on mxnet (#3581)
at that time , mkl pooling do not support Asymmetric padding filled by zeros, by default pool_enum::kValid will use mxnet pooling implmenetation